examples/index.html

```diff
@@ -56,6 +56,7 @@ <h2>Examples</h2>
                             <li><a href="2.1.5.1.5-facet-pivot.php">2.1.5.1.5 Facet pivot</a></li>
                             <li><a href="2.1.5.1.6-facet-interval.php">2.1.5.1.6 Facet interval</a></li>
                             <li><a href="2.1.5.1.7-facet-json-terms.php">2.1.5.1.7 Facet json terms</a></li>
+                            <li><a href="2.1.5.1.8-facet-json-query">2.1.5.1.8 Facet json query</a></li>
```

Comment:
Can you add `.php` to the link? It depends on the webserver config if it'll work without the extension.

```suggestion
                            <li><a href="2.1.5.1.8-facet-json-query.php">2.1.5.1.8 Facet json query</a></li>
```